models/topics.model.js

```diff
@@ -1,11 +1,14 @@
 const db = require("../db/connection");
+const fs = require("fs/promises");
 
 exports.fetchTopics = async () => {
   let queryStr = `SELECT * FROM topics`;
   let queryVals = [];
-
   const result = await db.query(queryStr, queryVals);
-  console.log('you are here')
-
   return result.rows;
 };
+exports.fetchEndpoints = async () => {
+  const result = await fs.readFile(`${__dirname}/../endpoints.json`, "utf-8");
+  const parsedResult = JSON.parse(result);
+  return parsedResult;
+};
```

Comment:
Models traditionally are for interacting with the database. Where are the endpoints in the database? 
Consider this function, you are reading a json file to a string, parsing it back to an object and then sending a json object back to the client in the controller. Could this chain of conversions be shortened?